subsystems/drivesubsystem.py

```diff
@@ -311,11 +310,7 @@ def __init__(self) -> None:
 
         # Create the gyro, a sensor which can indicate the heading of the robot relative
         # to a customizable position.
-        self.gyro = Pigeon2(constants.kPigeonCANId, constants.kCANivoreName)
-
-        toApply = Pigeon2Configuration()
-        self.gyro.configurator.apply(toApply)
-        self.gyro.get_yaw().set_update_frequency(100)
+        self.gyro = AHRS.create_spi()
```

Comment:
Fine but we should fix when pigeons come in

Reply:
agreed, its a revertable commit